colossalai/inference/config.py

```diff
@@ -393,3 +393,64 @@ class ModelShardInferenceConfig:
     use_cuda_kernel: bool = False
     use_spec_dec: bool = False
     use_flash_attn: bool = False
+
+
+@dataclass
+class GenerationParams:
+    """
+    Param for diffusion
+    """
+
+    # prompt: Union[str, List[str]] = None # NOTE no sure if we should add this param
+    prompt_2: Optional[Union[str, List[str]]] = None
+    prompt_3: Optional[Union[str, List[str]]] = None
```

Comment:
I'm a bit confused with these attributes. The purpose of `GenerationParams` is to provide default generation configs for diffusion models, right? Then is it appropriate to have default prompts for the second and the third tokenizers, or are there extra explanations/descriptions for these two prompts?